apm-webapp/pom.xml

```diff
@@ -30,14 +30,17 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <spring.boot.version>1.5.11.RELEASE</spring.boot.version>
+        <spring.boot.version>1.5.22.RELEASE</spring.boot.version>
```

Comment:
Related version in `dist-material/release-docs/LICENSE` is not updated.

Because `check-LICENSE.sh` doesn't check webapp, please manually check this

Reply:
done

Reply:
What I mean is 

https://github.com/apache/skywalking/blob/ca155dee0593714b6b93a1cc5d50ba49dd397f15/dist-material/release-docs/LICENSE#L254

... which is not updated

Reply:
done

Reply:
I thought what you mentioned is the embed tomcat.

Reply:
Spring boot 1.x has a series of CVE, that causes to upgrade embed tomcat separately. We should upgrade to Spring boot 2.x or switch to nodejs server ASAP.

Reply:
Let's open an issue to track this, we are not facing heavy traffic for web app, I think a NodeJS server should be better? Let's ping the UI team on that issue.

Reply:
Track in https://github.com/apache/skywalking/issues/7122